src/coreclr/vm/fcall.h

```diff
@@ -789,13 +300,6 @@ void HCallAssert(void*& cache, void* target);
 // the managed routine this code is associted with (for managed stack traces). This is passed with the
 // FC_INNER_PROLOG macro.
 //
-// The helper can set up a HELPER_METHOD_FRAME, but should pass the
-// Frame::FRAME_ATTR_EXACT_DEPTH|Frame::FRAME_ATTR_CAPTURE_DEPTH_2 which indicates the exact number of
-// unwinds to do to get back to managed code. Currently we only support depth 2 which means that the
-// HELPER_METHOD_FRAME needs to be set up in the function directly called by the FCALL. The helper should
-// use the NOINLINE macro to prevent the compiler from inlining it into the FCALL (which would obviously
-// mess up the unwind count).
-//
 // The other invariant that needs to hold is that the epilog walker needs to be able to get from the call to
```

Comment:
There is no epilog walker (gms...,h) and `FC_INNER_RETURN` anymore. This comment should be deleted.